The job **`aarch64-gnu`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/8677770237/job/23793852264) [(plain)](https://github.com/rust-lang-ci/rust/commit/9678dacddf5b68c03ce74e795f5a8d3f84afca82/checks/23793852264/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
 --enable-profiler \
 --enable-compiler-docs
# FIXME: Skipping cargo panic_abort_doc_tests due to https://github.com/rust-lang/rust/issues/123733
ENV SCRIPT python3 ../x.py --stage 2 test && \
  python3 ../x.py --stage 2 test src/tools/cargo --test-args "--skip panic_abort_doc_tests"

# ignore-tidy-linelength

set -ex
---
+ python3 ../x.py --stage 2 test src/tools/cargo --test-args --skip panic_abort_doc_tests
##[group]Building bootstrap
    Finished `dev` profile [unoptimized] target(s) in 0.10s
##[endgroup]
ERROR: no `test` rules matched ["panic_abort_doc_tests"]
HELP: run `x.py test --help --verbose` to show a list of available paths
NOTE: if you are adding a new Step to bootstrap itself, make sure you register it with `describe!`
  local time: Sun Apr 14 03:14:04 UTC 2024
  network time: Sun, 14 Apr 2024 03:14:04 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/src/ci/docker/host-aarch64/aarch64-gnu/Dockerfile
+++ b/src/ci/docker/host-aarch64/aarch64-gnu/Dockerfile
@@ -28,4 +28,4 @@ ENV RUST_CONFIGURE_ARGS \
  --enable-compiler-docs
 # FIXME: Skipping cargo panic_abort_doc_tests due to https://github.com/rust-lang/rust/issues/123733
 ENV SCRIPT python3 ../x.py --stage 2 test && \
-  python3 ../x.py --stage 2 test src/tools/cargo --test-args "--skip panic_abort_doc_tests"
+  python3 ../x.py --stage 2 test src/tools/cargo --test-args \"--skip panic_abort_doc_tests\"

